detectable/src/test/java/com/synopsys/integration/detectable/detectables/bitbake/functional/BuildFileFinderTest.java

```diff
@@ -0,0 +1,50 @@
+package com.synopsys.integration.detectable.detectables.bitbake.functional;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.File;
+
+import org.junit.jupiter.api.Test;
+
+import com.synopsys.integration.common.util.finder.SimpleFileFinder;
+import com.synopsys.integration.detectable.annotations.FunctionalTest;
+import com.synopsys.integration.detectable.detectables.bitbake.BuildFileFinder;
+import com.synopsys.integration.detectable.detectables.bitbake.model.BitbakeEnvironment;
+import com.synopsys.integration.exception.IntegrationException;
+
+@FunctionalTest
+public class BuildFileFinderTest {
+
+    @Test
+    void testDefault() throws IntegrationException {
+        BuildFileFinder finder = new BuildFileFinder(new SimpleFileFinder());
+        File buildDir = new File("src/test/resources/detectables/functional/bitbake/builddir_default");
```

Comment:
```suggestion
        File buildDir = FunctionalTestFiles.asFile("/bitbake/builddir_default");
```
This suggestion applies to all the methods in this test.